### PR TITLE
Navigation implementation #127

### PR DESCRIPTION
**Pull request by katibest**
Added sidebar navigation using boostrap. 
screen shot 2018-04-20 at 4 17 19 pm

### CODE REVIEW

File: apps/explorer_web/lib/explorer_web/templates/layout/app.html.eex [hunk 1]
```diff
@@ -6,15 +6,92 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title><%= gettext "POA Network Explorer" %></title>
     <link rel="stylesheet" href="<%= static_path(@conn, "/css/app.css") %>">
+    <!-- Bootstrap core CSS-->
+    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+    <!-- Custom fonts for this template-->
+    <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+    <!-- Custom styles for this template-->
+    <link href="css/sb-admin.css" rel="stylesheet">
```
Comment (jimmay5469) [inline apps/explorer_web/lib/explorer_web/templates/layout/app.html.eex]: I think this needs removed, you were pulling this out, right?

File: apps/explorer_web/lib/explorer_web/templates/layout/app.html.eex [hunk 2]
```diff
@@ -6,15 +6,92 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title><%= gettext "POA Network Explorer" %></title>
     <link rel="stylesheet" href="<%= static_path(@conn, "/css/app.css") %>">
+    <!-- Bootstrap core CSS-->
+    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
```
Comment (jimmay5469) [inline apps/explorer_web/lib/explorer_web/templates/layout/app.html.eex]: I don't think we should need this, I think you're including everything you need already via `app.scss`.

File: apps/explorer_web/lib/explorer_web/templates/layout/app.html.eex [hunk 3]
```diff
@@ -6,15 +6,92 @@
...
+    <!-- Custom fonts for this template-->
+    <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
```
Comment (jimmay5469) [inline apps/explorer_web/lib/explorer_web/templates/layout/app.html.eex]: I need to get this pulled in for you via npm. I'll take a look before we go to merge this.

File: apps/explorer_web/lib/explorer_web/templates/layout/app.html.eex [hunk 4]
```diff
@@ -6,15 +6,92 @@
...
+    <!-- Custom styles for this template-->
+    <link href="css/sb-admin.css" rel="stylesheet">
   </head>
+
   <body>
-    <%= render ExplorerWeb.LayoutView, "_header.html", assigns %>
```
Comment (jimmay5469) [inline apps/explorer_web/lib/explorer_web/templates/layout/app.html.eex]: @tmecklem can we use this render signature for shared templates that need a view backing them?

File: apps/explorer_web/lib/explorer_web/templates/layout/app.html.eex [hunk 5]
```diff
@@ -6,15 +6,92 @@
...
+                 $(this).toggleClass('active');
+             });
+         });
+      </script>
+      <script type='text/javascript' src='https://cdnjs.cloudflare.com/ajax/libs/fastclick/1.0.6/fastclick.js'></script>
+      <script type='text/javascript'>
+        $(function() {
+          FastClick.attach(document.body);
+        });
+      </script>
```
Comment (jimmay5469) [inline apps/explorer_web/lib/explorer_web/templates/layout/app.html.eex]: Was adding FastClick intentional? I'm not sure that we want it.

File: apps/explorer_web/assets/static/images/caret-down.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 345"><defs><style>.cls-1{fill:#bac4cb;}</style></defs><title>caret-down</title><path class="cls-1" d="M8,34.437c2.285-5.214,4.036-10.928,6.955-15.548C23.938,4.667,41.835,2.014,53.1,13.649c8.177,8.446,15.536,18.08,23.138,27.333Q162.282,145.7,248.28,250.476c.987,1.2,2.038,2.324,3.492,3.974,5.768-7.161,11.326-14.176,16.985-21.068q88.21-107.436,176.422-214.869c6.364-7.759,13.709-12.494,22.8-11.958,12.032.709,21.514,9.741,25.5,23.638.417,1.453,1.007,2.831,1.517,4.244V46.752c-3.546,16.083-14.345,25.392-23.2,36.419-15.007,18.7-30.3,37.059-45.486,55.54q-24.432,29.727-48.894,59.419-22.508,27.4-44.976,54.843-22.506,27.4-45.065,54.725c-6.142,7.427-12.228,14.947-18.666,21.982-9.086,9.93-25.413,10.143-34.4-.018-11.646-13.176-22.821-26.973-34.108-40.614-13.587-16.422-27.079-32.961-40.613-49.449q-22.5-27.409-45-54.816T69.578,130.009c-14.909-18.157-29.913-36.2-44.637-54.579A232.761,232.761,0,0,1,10.824,54.942c-1.44-2.259-1.91-5.434-2.824-8.19Q8,40.594,8,34.437Z"/></svg>
```
[the remark this answers is not in the document]
Comment (katibest) [reply apps/explorer_web/assets/static/images/caret-down.svg]: Nope!

**Comment by jimmay5469** [comment]
There was tested code which was no longer needed so we removed the tests and the code, which resulted in a .03% decrease in coverage and a coveralls failure. I am going to proceed with the merge because the failure was not a result of adding untested code.

**Pull request merged**